src/components/DailyVideo.tsx

```diff
@@ -182,14 +182,19 @@ export const DailyVideo = forwardRef<HTMLVideoElement, Props>(
               });
             }
           });
-        };
+        }
 
         handleResize();
-        video?.addEventListener('resize', handleResize);
+        video.addEventListener('loadedmetadata', handleResize);
+        video.addEventListener('resize', handleResize);
 
-        return () => video?.removeEventListener('resize', handleResize);
+        return () => {
+          if (frame) cancelAnimationFrame(frame);
+          video.removeEventListener('loadedmetadata', handleResize);
+          video.removeEventListener('resize', handleResize);
+        };
       },
-      [onResize, videoTrack]
```

Comment:
I removed `videoTrack` because I think the `loadedmetadata` and `resize` listeners should handle this without re-running the effect.